src/components/RouteStatistics.tsx

```diff
@@ -12,16 +11,14 @@ const formatEngagement = (timeline?: TimelineStatistics): string | undefined =>
   return `${(100 * (engagedDuration / duration)).toFixed(0)}%`
 }
 
-const RouteStatistics: VoidComponent<{ class?: string; route?: Route }> = (props) => {
-  const [timeline] = createResource(() => props.route, getTimelineStatistics)
-
+const RouteStatistics: VoidComponent<{ class?: string; route?: Route; timeline?: TimelineStatistics }> = (props) => {
```

Comment:
We use this component in `<RouteActivity>` too, but because the prop is optional there is no error

Reply:
```suggestion
const RouteStatistics: VoidComponent<{ class?: string; route: Route | undefined; timeline: TimelineStatistics | undefined }> = (props) => {

Reply:
I think that would cause an error to be raised?